virttest/libvirt_installer.py

```diff
@@ -0,0 +1,125 @@
+"""
+Installer code that implement KVM specific bits.
+
+See BaseInstaller class in base_installer.py for interface details.
+"""
+
+import os
+import platform
+from avocado.utils import process
+import base_installer
```

Comment:
one line space between builtin, framework packages,

It is better to import base_installer as,
```python
from . import base_installer
```